pkg/planner/util/fixcontrol/get.go

```diff
@@ -11,6 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
```

Comment:
I guess we should put this above those `fix` constants instead of the license header.